packages/@aws-cdk/cfnspec/spec-source/540_SSM_Association_Parameters_patch.json

```diff
@@ -0,0 +1,20 @@
+{
+  "ResourceTypes": {
+    "AWS::SSM::Association": {
+      "patch": {
+        "description": "Removes 'ItemType' property since 'ParameterValues' is undefined and specifies it to be a list of String",
```

Comment:
```suggestion
        "description": "Removes 'ItemType' property since 'ParameterValues' is (currently) not defined in the spec and the documentation states it to be a list of String",
```